docs/docs/environment-variables.md

```diff
@@ -117,7 +156,11 @@ module.exports = {
 }
 ```
 
-Local testing of staging is as simple as
+This will then load the values from the relevant environment's `.env.*` file and use them in the graphql database and the analytics plugin respectively.
```

Comment:
```suggestion
This will then load the values from the relevant environment's `.env.*` file and make them available via GraphQL queries and the analytics plugin respectively.
```